bothan-core/src/telemetry/registry.rs

```diff
@@ -0,0 +1,28 @@
+use std::sync::Arc;
+
+use opentelemetry::global;
+use opentelemetry_sdk::{metrics::{MetricError, SdkMeterProvider}, Resource};
+use prometheus::Registry;
+
+const SERVICE_NAME: &str = "bothan";
+
+pub fn init_telemetry_registry() -> Result<Arc<Registry>, MetricError> {
+    let registry = Arc::new(Registry::new());
```

Comment:
registry is already send+sync (internally wrapped as arc) we dont need to rewrap it with arc